test/utilities/extensionUtils.ts

```diff
@@ -141,10 +142,20 @@ export async function showRunningExtensions(): Promise<void> {
   await utilities.executeQuickPick('Developer: Show Running Extensions');
   await browser.waitUntil(
     async () => {
-      const runningExtensionsTab = await $(
-        "//div[contains(@class, 'active') and contains(@class, 'selected') and .//*[contains(text(), 'Running Extensions')]]"
-      );
-      return runningExtensionsTab.isDisplayed();
+      let runningExtensionsTab;
+      if (
+        EnvironmentSettings.getInstance().vscodeVersion === 'stable' ||
+        semver.gte(EnvironmentSettings.getInstance().vscodeVersion, '1.90.0')
+      ) {
+        runningExtensionsTab = await $(
+          "//div[contains(@class, 'active') and contains(@class, 'selected') and .//*[contains(text(), 'Running Extensions')]]"
+        );
+      } else {
+        runningExtensionsTab = await $(
+          "//div[contains(@class, 'monaco-list-row') and .//*[contains(text(), 'Running Extensions')]]"
+        );
+      }
+      return (await runningExtensionsTab.getTitle()).includes('Running Extensions');
```

Comment:
`isDisplayed()` doesn't work for 1.86 so I changed it to check for whether the title of the VSCode window contains "Running Extensions".  If the "Running Extensions" tab is the one that is opened, the title of the VSCode window will be "[Extension Development Host] Running Extensions – \<project name\>".